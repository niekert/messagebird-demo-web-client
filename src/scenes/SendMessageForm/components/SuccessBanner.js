import React from 'react';
import styled from 'styled-components';
import { prop } from 'styled-tools';
import Banner from 'shared/components/Banner';

const EmojiWrapper = styled.span`
  font-size: 25px;
  margin-right: ${prop('theme.spacing.0')};
`;

const SuccessBanner = () => (
  <Banner>
    <EmojiWrapper>
      <span role="img" aria-label="success">
        ðŸ˜Ž
      </span>
    </EmojiWrapper>
    Your message was sent successfully
  </Banner>
);

export default SuccessBanner;
