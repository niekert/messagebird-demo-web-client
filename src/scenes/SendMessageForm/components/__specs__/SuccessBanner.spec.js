import React from 'react';
import { shallow } from 'enzyme';
import SuccessBanner from '../SuccessBanner';

describe('<SuccessBanner />', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<SuccessBanner />);
    expect(wrapper).toMatchSnapshot();
  });
});
