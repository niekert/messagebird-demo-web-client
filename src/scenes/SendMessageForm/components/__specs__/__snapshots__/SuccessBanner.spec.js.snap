// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SuccessBanner /> renders correctly 1`] = `
<Banner
  className={null}
  durationMs={3000}
  type="OK"
>
  <SuccessBanner__EmojiWrapper>
    <span
      aria-label="success"
      role="img"
    >
      ðŸ•Š
    </span>
  </SuccessBanner__EmojiWrapper>
  Your message was sent successfully
</Banner>
`;
